docs/catalog/minimizing-deployed-environments.md

```diff
@@ -0,0 +1,37 @@
+# Minimising the total number of deployed environments
+
+## Version
+1.0
+
+## Submitted By
+Cameron Casher (@camcash17)
+
+## Published Date
+TBD
+
+## Intent
+Minimising the total number of deployed environments for an application.
+
+## Tags
+Cloud, Environments, Production, Development, Performance, QA
+
+## Problem
+From an energy-efficiency perspective, it's better to minimise the amount of deployed environments for an application so that less energy is required because there is less resources provisioned.
+
+## Solution
+Minimise the amount of deployed environments, perhaps considering the necessity of a QA or Performance testing environment.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+Concerning the SCI equation. Reducing the amount of deployed environments will impact one part:
+
+- `E`: We reduce the total electricity required by reducing provisioned resources.
+- `M`: By reducing the amount of nodes running on underlying machines, we reduce the total embodied carbon.
```

Comment:
I think this should just focus on reducing the total number of machines and resources. Deployed environments don't typically reduce nodes but rather full machines/virtual machines.